stock_quant_reserved_qty_uom/tests/test_stock_quant_reserved_qty_uom.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp.tests.common import TransactionCase
+
+
+class TestStockQuantReservedQtyUom(TransactionCase):
```

Comment:
@jbeficent It seems that assert test is not executed (if I understand codecoverage result). Can you check ?